src/outputs/output_utils.hpp

```diff
@@ -216,31 +290,35 @@ std::vector<T> FlattenBlockInfo(Mesh *pm, int shape, Function_t f) {
 
 // mirror must be provided because copying done externally
 template <typename Data_t, typename idx_t, typename Function_t>
-void PackOrUnpackVar(MeshBlock *pmb, Variable<Real> *pvar, bool do_ghosts, idx_t &idx,
-                     std::vector<Data_t> &data, Function_t f) {
-  const auto &Nt = pvar->GetDim(6);
-  const auto &Nu = pvar->GetDim(5);
-  const auto &Nv = pvar->GetDim(4);
+void PackOrUnpackVar(const VarInfo &info, Variable<Real> *pvar, bool do_ghosts,
+                     idx_t &idx, std::vector<Data_t> &data, Function_t f) {
```

Comment:
Are `data` and `pvar` necessary here? Looks like they might actually be accessed by lambda capture.

Reply:
You're right! Will remove.